src/components/Comments.jsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+import CommentCard from "./CommentCard";
+import Button from "./Button";
+
+export default function Comments(props)
+{
+    const { article_id } = props;
+
+    const [comments, setComments] = useState([]);
+    const [showAllComments, setShowAllComments] = useState(false);
+    
+    const commentsCount = comments.length;
+
+    useEffect(() => {
+        const fetchComments = async () =>
+        {
+            try
+            {
+                const response = await axios.get(`https://nc-news-egcd.onrender.com/api/articles/${article_id}/comments`);
```

Comment:
would highly highly recommend creating a api.js file now to keep your API calls in one place, make use of the baseURL